sam-helm-charts/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: sam
+description: A Helm chart for Kubernetes
```

Comment:
Can you add a short description of what this Helm chart helps you to install here?